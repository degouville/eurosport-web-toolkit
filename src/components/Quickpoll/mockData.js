export const choices = [
  { id: 123451, displayText: 'Nadal' },
  { id: 123452, displayText: 'Djokovic' },
  { id: 123453, displayText: 'Zverev' },
  { id: 123454, displayText: 'Federer' },
  { id: 123455, displayText: 'Other' },
];

export const choicesWithResults = [
  { id: 123451, displayText: 'Nadal', result: '50%' },
  { id: 123452, displayText: 'Djokovic', result: '47%' },
  { id: 123453, displayText: 'Zverev', result: '34%' },
  { id: 123454, displayText: 'Federer', result: '16%' },
  { id: 123455, displayText: 'Other', result: '1%' },
];
