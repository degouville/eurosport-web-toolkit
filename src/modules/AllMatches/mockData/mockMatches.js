const liveMatch = {
  isLive: true,
  isWatchable: true,
  matchUrl: 'https://eurosport.com',
  data: {
    topTeam: {
      playerOneName: 'MENENDEZ-MACEIRAS',
      sets: [
        { set: 1, score: 6, won: true, tie: null },
        { set: 2, score: 6, won: true, tie: null },
        { set: 3, score: 6, won: false, tie: null },
        { set: 4, score: 6, won: true, tie: null },
        { set: 5, score: 3, won: false, tie: null },
      ],
      isServing: true,
    },
    bottomTeam: {
      playerOneName: 'DAVIDOVICH FOKINA',
      sets: [
        { set: 1, score: 3, won: false, tie: null },
        { set: 2, score: 3, won: false, tie: null },
        { set: 3, score: 7, won: true, tie: null },
        { set: 4, score: 3, won: false, tie: null },
        { set: 5, score: 6, won: true, tie: null },
      ],
      isServing: false,
    },
  },
};

const upcomingMatch = {
  isLive: false,
  isWatchable: false,
  matchUrl: 'https://eurosport.com',
  data: {
    topTeam: {
      playerOneName: 'MENENDEZ-MACEIRAS',
      sets: [],
      isServing: false,
    },
    bottomTeam: {
      playerOneName: 'DAVIDOVICH FOKINA',
      sets: [],
      isServing: false,
    },
  },
};

const finishedMatch = {
  isLive: false,
  isWatchable: false,
  matchUrl: 'https://eurosport.com',
  data: {
    topTeam: {
      playerOneName: 'RAHMAN MAKSSOUD',
      sets: [
        { set: 1, score: 6, won: true, tie: null },
        { set: 2, score: 6, won: true, tie: null },
        { set: 3, score: 6, won: false, tie: 2 },
        { set: 4, score: 6, won: true, tie: null },
        { set: 5, score: 3, won: false, tie: null },
      ],
      isServing: false,
      hasWon: true,
    },
    bottomTeam: {
      playerOneName: 'DAVIDOVICH FOKINA',
      sets: [
        { set: 1, score: 3, won: false, tie: null },
        { set: 2, score: 3, won: false, tie: null },
        { set: 3, score: 7, won: true, tie: 7 },
        { set: 4, score: 3, won: false, tie: null },
        { set: 5, score: 6, won: true, tie: null },
      ],
      isServing: false,
      hasWon: false,
    },
  },
};

export const liveMatches = new Array(8).fill().map((el, index) => ({ id: index, ...liveMatch }));

export const upcomingMatches = new Array(5).fill().map((el, index) => ({ id: index, ...upcomingMatch }));

export const finishedMatches = new Array(8).fill().map((el, index) => ({ id: index, ...finishedMatch }));
