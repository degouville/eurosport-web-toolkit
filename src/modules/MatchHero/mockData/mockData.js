export const afterLabels = [
  {
    text: 'French Open',
    bold: true,
    color: 'white',
  },
  {
    text: "Men's singles | Final",
    color: 'blue',
  },
  {
    text: 'Result',
    color: 'cyan',
  },
];

export const duringLabels = [
  {
    text: 'French Open',
    bold: true,
    color: 'white',
  },
  {
    text: "Men's singles | Final",
    color: 'blue',
  },
  {
    text: 'Live',
    color: 'red',
    icon: 'â€¢',
  },
];

export const beforeLabels = [
  {
    text: 'French Open',
    bold: true,
    color: 'white',
  },
  {
    text: "Men's singles | Final",
    color: 'blue',
  },
];

export const scoreDataDuring = {
  topTeam: {
    playerOneName: 'A. MENENDEZ-MACEIRAS',
    playerTwoName: 'A. Rahman MAKSSOUD',
    sets: [
      {
        set: 1,
        score: 6,
        won: true,
        tie: null,
      },
      {
        set: 2,
        score: 6,
        won: true,
        tie: null,
      },
      {
        set: 3,
        score: 6,
        won: false,
        tie: 2,
      },
      {
        set: 4,
        score: 6,
        won: true,
        tie: null,
      },
      {
        set: 5,
        score: 3,
        won: false,
        tie: null,
      },
    ],
    hasWon: null,
    isServing: false,
  },
  bottomTeam: {
    playerOneName: 'A. MENENDEZ-MACEIRAS',
    playerTwoName: 'A. DAVIDOVICH FOKINA',
    sets: [
      {
        set: 1,
        score: 3,
        won: false,
        tie: null,
      },
      {
        set: 2,
        score: 3,
        won: true,
        tie: null,
      },
      {
        set: 3,
        score: 7,
        won: true,
        tie: 7,
      },
      {
        set: 4,
        score: 3,
        won: false,
        tie: null,
      },
      {
        set: 5,
        score: 5,
        won: false,
        tie: null,
      },
    ],
    hasWon: null,
    isServing: true,
  },
};

export const scoreDataAfter = {
  ...scoreDataDuring,
  topTeam: {
    ...scoreDataDuring.topTeam,
    hasWon: false,
  },
  bottomTeam: {
    ...scoreDataDuring.bottomTeam,
    sets: [
      ...scoreDataDuring.bottomTeam.sets.slice(0, 4),
      {
        set: 5,
        score: 7,
        won: true,
        tie: null,
      },
    ],
    isServing: false,
    hasWon: true,
  },
};
