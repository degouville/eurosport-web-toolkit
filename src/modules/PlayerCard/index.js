import React from 'react';
import PropTypes from 'prop-types';
import styled, { css } from 'react-emotion';
import * as colors from '../../colors';
import * as breakpoints from '../../breakpoints';
import { fontFamilies } from '../../typography';
import PlayerInfos, { playerType } from '../../components/PlayerInfos';
import ScoreBlock, { teamDataType } from '../../components/ScoreBlock';
import ViewMore, { StyledButton } from '../ViewMore';

const StyledContainer = styled.div`
  background-color: ${colors.ebonyClay};
  padding-bottom: 45px;
`;

export const StyledBackground = styled.div`
  display: block;
  position: absolute;
  width: 100%;
  height: 100%;
  top: -45px;
  max-height: 150px;
  background-size: cover;
  background-repeat: no-repeat;
  ${props => `background-image: url(${props.backgroundImageUrl});`}
`;

const StyledPlayerInfos = styled(PlayerInfos)`
  margin-top: 30px;
`;

const StyledPreviousMatches = styled.div`
  font-family: ${fontFamilies.interUi};
  font-size: 12px;
  font-weight: 600;
  letter-spacing: 1px;
  line-height: 18px;
  color: ${colors.coreNeutral3};
  text-transform: uppercase;
  margin: 21px 31px;
`;

export const StyledScoreBlock = styled.div`
  display: inherit;
`;

const StyledBefore = styled.div`
  :before {
    content: '';
    width: 41px;
    height: 20px;
    border-right: 2px solid ${colors.coreLightMinus1};
    display: block;
  }
`;

const StyledMatches = styled.div`
  text-align: center;
  ${breakpoints.medium(css`
    text-align: left;
  `)};
  ul {
    margin-bottom: 45px;
  }
  ${StyledButton} {
    ${breakpoints.medium(css`
      margin-left: 70px;
    `)};
  }
`;

const PlayerCard = ({ playerInfo, backgroundImageUrl, previousMatches }) => (
  <StyledContainer>
    <StyledBackground backgroundImageUrl={backgroundImageUrl} />
    <StyledPlayerInfos player={playerInfo} />

    {previousMatches && previousMatches.length > 0 && (
      <>
        <StyledPreviousMatches>PREVIOUS MATCHES</StyledPreviousMatches>
        <StyledMatches>
          <ViewMore showLessText="View Less Matches" showMoreText="View More Matches">
            {previousMatches.map((match, index) => (
              <StyledScoreBlock key={match.id}>
                {index > 0 && <StyledBefore />}
                <ScoreBlock.ScoreBlock
                  matchUrl={match.matchUrl}
                  data={match.data}
                  isLive={false}
                  isWatchable={false}
                  displayLeftCircle={match.hasWon ? 'won' : 'lost'}
                />
              </StyledScoreBlock>
            ))}
          </ViewMore>
        </StyledMatches>
      </>
    )}
  </StyledContainer>
);

const matchType = PropTypes.shape({
  id: PropTypes.number.isRequired,
  matchUrl: PropTypes.string.isRequired,
  data: PropTypes.shape({
    bottomTeam: teamDataType,
    topTeam: teamDataType,
  }).isRequired,
  hasWon: PropTypes.bool,
});

PlayerCard.propTypes = {
  playerInfo: playerType.isRequired,
  backgroundImageUrl: PropTypes.string.isRequired,
  previousMatches: PropTypes.arrayOf(matchType).isRequired,
};

export default PlayerCard;
