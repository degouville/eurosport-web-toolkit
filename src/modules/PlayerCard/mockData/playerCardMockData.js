export const BACKGROUND_IMG = 'http://dnl.eurosport.com/web-toolkit/playercard/bg1.png';

export const playerInfo = {
  firstName: 'Rafael',
  lastName: 'Nadal',
  age: '32',
  height: '1.85',
  weight: '85',
  country: 'Spain',
  flagUrl: 'https://i.eurosport.com/_iss_/geo/country/flag/medium/2203.png',
  ranking: '2',
  competition: 'ATP',
  pictureUrl: 'https://i.eurosport.com/_iss_/person/pp_clubteam/large/435121.jpg',
};

export const previousMatches = [
  {
    id: 123451,
    matchUrl: 'https://eurosport.com',
    hasWon: true,
    labels: [{ text: 'ROLAND-GARROS' }, { text: 'SINGLES' }, { text: 'FINAL' }],
    data: {
      topTeam: {
        playerOneName: 'Nadal',
        sets: [
          { set: 1, score: 6, won: true, tie: null },
          { set: 2, score: 6, won: true, tie: null },
          { set: 3, score: 6, won: false, tie: 2 },
          { set: 4, score: 6, won: true, tie: null },
          { set: 5, score: 3, won: false, tie: null },
        ],
        hasWon: true,
        isServing: false,
      },
      bottomTeam: {
        playerOneName: 'Federer',
        sets: [
          { set: 1, score: 3, won: false, tie: null },
          { set: 2, score: 3, won: true, tie: null },
          { set: 3, score: 7, won: true, tie: 7 },
          { set: 4, score: 3, won: false, tie: null },
          { set: 5, score: 6, won: true, tie: null },
        ],
        hasWon: false,
        isServing: false,
      },
    },
  },
  {
    id: 123452,
    matchUrl: 'https://eurosport.com',
    hasWon: true,
    labels: [{ text: 'ROLAND-GARROS' }, { text: 'DOUBLES' }, { text: '2ND ROUND' }],
    data: {
      topTeam: {
        playerOneName: 'Nadal',
        sets: [
          { set: 1, score: 6, won: true, tie: null },
          { set: 2, score: 6, won: true, tie: null },
          { set: 3, score: 6, won: false, tie: 2 },
          { set: 4, score: 6, won: true, tie: null },
          { set: 5, score: 3, won: false, tie: null },
        ],
        hasWon: true,
        isServing: false,
      },
      bottomTeam: {
        playerOneName: 'Federer',
        sets: [
          { set: 1, score: 3, won: false, tie: null },
          { set: 2, score: 3, won: true, tie: null },
          { set: 3, score: 7, won: true, tie: 7 },
          { set: 4, score: 3, won: false, tie: null },
          { set: 5, score: 6, won: true, tie: null },
        ],
        hasWon: false,
        isServing: false,
      },
    },
  },
  {
    id: 123453,
    matchUrl: 'https://eurosport.com',
    hasWon: false,
    labels: [{ text: 'ROLAND-GARROS' }, { text: 'SINGLES' }, { text: '1ST ROUND' }],
    data: {
      topTeam: {
        playerOneName: 'Nadal',
        sets: [
          { set: 1, score: 6, won: false, tie: null },
          { set: 2, score: 2, won: false, tie: null },
          { set: 3, score: 3, won: false, tie: null },
        ],
        hasWon: false,
        isServing: false,
      },
      bottomTeam: {
        playerOneName: 'Federer',
        sets: [
          { set: 1, score: 7, won: true, tie: null },
          { set: 2, score: 6, won: true, tie: null },
          { set: 3, score: 6, won: true, tie: null },
        ],
        hasWon: true,
        isServing: false,
      },
    },
  },
  {
    id: 123454,
    matchUrl: 'https://eurosport.com',
    hasWon: true,
    labels: [{ text: 'ROLAND-GARROS' }, { text: 'SINGLES' }, { text: '2ND ROUND' }],
    data: {
      topTeam: {
        playerOneName: 'Nadal',
        sets: [
          { set: 1, score: 6, won: true, tie: null },
          { set: 2, score: 6, won: true, tie: null },
          { set: 3, score: 6, won: false, tie: 2 },
          { set: 4, score: 6, won: true, tie: null },
          { set: 5, score: 3, won: false, tie: null },
        ],
        hasWon: true,
        isServing: false,
      },
      bottomTeam: {
        playerOneName: 'Federer',
        sets: [
          { set: 1, score: 3, won: false, tie: null },
          { set: 2, score: 3, won: true, tie: null },
          { set: 3, score: 7, won: true, tie: 7 },
          { set: 4, score: 3, won: false, tie: null },
          { set: 5, score: 6, won: true, tie: null },
        ],
        hasWon: false,
        isServing: false,
      },
    },
  },
  {
    id: 123455,
    matchUrl: 'https://eurosport.com',
    hasWon: true,
    labels: [{ text: 'ROLAND-GARROS' }, { text: 'SINGLES' }, { text: 'FINAL' }],
    data: {
      topTeam: {
        playerOneName: 'Nadal',
        sets: [
          { set: 1, score: 6, won: true, tie: null },
          { set: 2, score: 6, won: true, tie: null },
          { set: 3, score: 6, won: false, tie: 2 },
          { set: 4, score: 6, won: true, tie: null },
          { set: 5, score: 3, won: false, tie: null },
        ],
        hasWon: true,
        isServing: false,
      },
      bottomTeam: {
        playerOneName: 'Federer',
        sets: [
          { set: 1, score: 3, won: false, tie: null },
          { set: 2, score: 3, won: true, tie: null },
          { set: 3, score: 7, won: true, tie: 7 },
          { set: 4, score: 3, won: false, tie: null },
          { set: 5, score: 6, won: true, tie: null },
        ],
        hasWon: false,
        isServing: false,
      },
    },
  },
];
