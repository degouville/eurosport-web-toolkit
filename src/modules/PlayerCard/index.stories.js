import React from 'react';
import { storiesOf } from '@storybook/react';
import { object, text } from '@storybook/addon-knobs';
import PlayerCard from '.';

import { BACKGROUND_IMG, previousMatches, playerInfo } from './mockData/playerCardMockData';

const playerCardStory = storiesOf('Modules|PlayerCard', module);

playerCardStory.add('PlayerCard', () => (
  <PlayerCard
    playerInfo={object('playerInfo', playerInfo)}
    backgroundImageUrl={text('backgroundImageUrl', BACKGROUND_IMG)}
    previousMatches={object('previousMatches', previousMatches)}
  />
));
