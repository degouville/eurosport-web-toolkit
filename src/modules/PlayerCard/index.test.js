import React from 'react';
import { shallow, mount } from 'enzyme';
import { PlayerInfos } from '../..';
import PlayerCard, { StyledScoreBlock, StyledBackground } from '.';

import { BACKGROUND_IMG, previousMatches, playerInfo } from './mockData/playerCardMockData';

describe('PlayerCard', () => {
  it('should render as expected', () => {
    expect(
      shallow(
        <PlayerCard playerInfo={playerInfo} backgroundImageUrl={BACKGROUND_IMG} previousMatches={previousMatches} />
      )
    ).toMatchSnapshot();
  });

  it('should render the background image', () => {
    const wrapper = mount(
      <PlayerCard playerInfo={playerInfo} backgroundImageUrl={BACKGROUND_IMG} previousMatches={previousMatches} />
    );
    const backgroundImg = wrapper.find(StyledBackground);
    expect(backgroundImg).toHaveStyleRule('background-image', `url(${BACKGROUND_IMG})`);
  });

  it('should render PlayerInfos', () => {
    const wrapper = mount(
      <PlayerCard playerInfo={playerInfo} backgroundImageUrl={BACKGROUND_IMG} previousMatches={previousMatches} />
    );
    expect(wrapper.find(PlayerInfos)).toHaveLength(1);
  });

  it('should render all the previous matches', () => {
    const wrapper = mount(
      <PlayerCard playerInfo={playerInfo} backgroundImageUrl={BACKGROUND_IMG} previousMatches={previousMatches} />
    );
    const expectedLength = previousMatches.length;
    expect(wrapper.find(StyledScoreBlock)).toHaveLength(expectedLength);
  });
});
