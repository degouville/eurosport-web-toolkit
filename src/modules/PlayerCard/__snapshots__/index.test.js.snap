// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerCard should render as expected 1`] = `
<Styled(div)>
  <Styled(div)
    backgroundImageUrl="http://dnl.eurosport.com/web-toolkit/playercard/bg1.png"
  />
  <Styled(PlayerInfos)
    ageText="Age"
    heightText="Height (m)"
    player={
      Object {
        "age": "32",
        "competition": "ATP",
        "country": "Spain",
        "firstName": "Rafael",
        "flagUrl": "https://i.eurosport.com/_iss_/geo/country/flag/medium/2203.png",
        "height": "1.85",
        "lastName": "Nadal",
        "pictureUrl": "https://i.eurosport.com/_iss_/person/pp_clubteam/large/435121.jpg",
        "ranking": "2",
        "weight": "85",
      }
    }
    rankingText="Ranking"
    weightText="Weight (Kg)"
  />
  <Styled(div)>
    Previous matches
  </Styled(div)>
  <Styled(div)>
    <ViewMore
      animationDelay={400}
      forceToggle={false}
      onClick={null}
      showLessItemCount={1}
      showLessText="Show less matches"
      showMoreText="Show more matches"
    >
      <Styled(div)
        key="123451"
      >
        <Styled(div)>
          <Labels
            isSimpleMode={true}
            labels={
              Array [
                Object {
                  "text": "ROLAND-GARROS",
                },
                Object {
                  "text": "SINGLES",
                },
                Object {
                  "text": "FINAL",
                },
              ]
            }
          />
        </Styled(div)>
        <ScoreBlock
          data={
            Object {
              "bottomTeam": Object {
                "hasWon": false,
                "isServing": false,
                "playerOneName": "Federer",
                "sets": Array [
                  Object {
                    "score": 3,
                    "set": 1,
                    "tie": null,
                    "won": false,
                  },
                  Object {
                    "score": 3,
                    "set": 2,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 7,
                    "set": 3,
                    "tie": 7,
                    "won": true,
                  },
                  Object {
                    "score": 3,
                    "set": 4,
                    "tie": null,
                    "won": false,
                  },
                  Object {
                    "score": 6,
                    "set": 5,
                    "tie": null,
                    "won": true,
                  },
                ],
              },
              "topTeam": Object {
                "hasWon": true,
                "isServing": false,
                "playerOneName": "Nadal",
                "sets": Array [
                  Object {
                    "score": 6,
                    "set": 1,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 6,
                    "set": 2,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 6,
                    "set": 3,
                    "tie": 2,
                    "won": false,
                  },
                  Object {
                    "score": 6,
                    "set": 4,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 3,
                    "set": 5,
                    "tie": null,
                    "won": false,
                  },
                ],
              },
            }
          }
          displayLeftCircle="won"
          hasButton={true}
          isLive={false}
          isWatchable={false}
          liveButtonText="Live"
          matchInfoButtonText="Match Info"
          matchUrl="https://eurosport.com"
          schedule={null}
        />
      </Styled(div)>
      <Styled(div)
        key="123452"
      >
        <Styled(div)>
          <Labels
            isSimpleMode={true}
            labels={
              Array [
                Object {
                  "text": "ROLAND-GARROS",
                },
                Object {
                  "text": "DOUBLES",
                },
                Object {
                  "text": "2ND ROUND",
                },
              ]
            }
          />
        </Styled(div)>
        <ScoreBlock
          data={
            Object {
              "bottomTeam": Object {
                "hasWon": false,
                "isServing": false,
                "playerOneName": "Federer",
                "sets": Array [
                  Object {
                    "score": 3,
                    "set": 1,
                    "tie": null,
                    "won": false,
                  },
                  Object {
                    "score": 3,
                    "set": 2,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 7,
                    "set": 3,
                    "tie": 7,
                    "won": true,
                  },
                  Object {
                    "score": 3,
                    "set": 4,
                    "tie": null,
                    "won": false,
                  },
                  Object {
                    "score": 6,
                    "set": 5,
                    "tie": null,
                    "won": true,
                  },
                ],
              },
              "topTeam": Object {
                "hasWon": true,
                "isServing": false,
                "playerOneName": "Nadal",
                "sets": Array [
                  Object {
                    "score": 6,
                    "set": 1,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 6,
                    "set": 2,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 6,
                    "set": 3,
                    "tie": 2,
                    "won": false,
                  },
                  Object {
                    "score": 6,
                    "set": 4,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 3,
                    "set": 5,
                    "tie": null,
                    "won": false,
                  },
                ],
              },
            }
          }
          displayLeftCircle="won"
          hasButton={true}
          isLive={false}
          isWatchable={false}
          liveButtonText="Live"
          matchInfoButtonText="Match Info"
          matchUrl="https://eurosport.com"
          schedule={null}
        />
      </Styled(div)>
      <Styled(div)
        key="123453"
      >
        <Styled(div)>
          <Labels
            isSimpleMode={true}
            labels={
              Array [
                Object {
                  "text": "ROLAND-GARROS",
                },
                Object {
                  "text": "SINGLES",
                },
                Object {
                  "text": "1ST ROUND",
                },
              ]
            }
          />
        </Styled(div)>
        <ScoreBlock
          data={
            Object {
              "bottomTeam": Object {
                "hasWon": true,
                "isServing": false,
                "playerOneName": "Federer",
                "sets": Array [
                  Object {
                    "score": 7,
                    "set": 1,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 6,
                    "set": 2,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 6,
                    "set": 3,
                    "tie": null,
                    "won": true,
                  },
                ],
              },
              "topTeam": Object {
                "hasWon": false,
                "isServing": false,
                "playerOneName": "Nadal",
                "sets": Array [
                  Object {
                    "score": 6,
                    "set": 1,
                    "tie": null,
                    "won": false,
                  },
                  Object {
                    "score": 2,
                    "set": 2,
                    "tie": null,
                    "won": false,
                  },
                  Object {
                    "score": 3,
                    "set": 3,
                    "tie": null,
                    "won": false,
                  },
                ],
              },
            }
          }
          displayLeftCircle="lost"
          hasButton={true}
          isLive={false}
          isWatchable={false}
          liveButtonText="Live"
          matchInfoButtonText="Match Info"
          matchUrl="https://eurosport.com"
          schedule={null}
        />
      </Styled(div)>
      <Styled(div)
        key="123454"
      >
        <Styled(div)>
          <Labels
            isSimpleMode={true}
            labels={
              Array [
                Object {
                  "text": "ROLAND-GARROS",
                },
                Object {
                  "text": "SINGLES",
                },
                Object {
                  "text": "2ND ROUND",
                },
              ]
            }
          />
        </Styled(div)>
        <ScoreBlock
          data={
            Object {
              "bottomTeam": Object {
                "hasWon": false,
                "isServing": false,
                "playerOneName": "Federer",
                "sets": Array [
                  Object {
                    "score": 3,
                    "set": 1,
                    "tie": null,
                    "won": false,
                  },
                  Object {
                    "score": 3,
                    "set": 2,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 7,
                    "set": 3,
                    "tie": 7,
                    "won": true,
                  },
                  Object {
                    "score": 3,
                    "set": 4,
                    "tie": null,
                    "won": false,
                  },
                  Object {
                    "score": 6,
                    "set": 5,
                    "tie": null,
                    "won": true,
                  },
                ],
              },
              "topTeam": Object {
                "hasWon": true,
                "isServing": false,
                "playerOneName": "Nadal",
                "sets": Array [
                  Object {
                    "score": 6,
                    "set": 1,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 6,
                    "set": 2,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 6,
                    "set": 3,
                    "tie": 2,
                    "won": false,
                  },
                  Object {
                    "score": 6,
                    "set": 4,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 3,
                    "set": 5,
                    "tie": null,
                    "won": false,
                  },
                ],
              },
            }
          }
          displayLeftCircle="won"
          hasButton={true}
          isLive={false}
          isWatchable={false}
          liveButtonText="Live"
          matchInfoButtonText="Match Info"
          matchUrl="https://eurosport.com"
          schedule={null}
        />
      </Styled(div)>
      <Styled(div)
        key="123455"
      >
        <Styled(div)>
          <Labels
            isSimpleMode={true}
            labels={
              Array [
                Object {
                  "text": "ROLAND-GARROS",
                },
                Object {
                  "text": "SINGLES",
                },
                Object {
                  "text": "FINAL",
                },
              ]
            }
          />
        </Styled(div)>
        <ScoreBlock
          data={
            Object {
              "bottomTeam": Object {
                "hasWon": false,
                "isServing": false,
                "playerOneName": "Federer",
                "sets": Array [
                  Object {
                    "score": 3,
                    "set": 1,
                    "tie": null,
                    "won": false,
                  },
                  Object {
                    "score": 3,
                    "set": 2,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 7,
                    "set": 3,
                    "tie": 7,
                    "won": true,
                  },
                  Object {
                    "score": 3,
                    "set": 4,
                    "tie": null,
                    "won": false,
                  },
                  Object {
                    "score": 6,
                    "set": 5,
                    "tie": null,
                    "won": true,
                  },
                ],
              },
              "topTeam": Object {
                "hasWon": true,
                "isServing": false,
                "playerOneName": "Nadal",
                "sets": Array [
                  Object {
                    "score": 6,
                    "set": 1,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 6,
                    "set": 2,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 6,
                    "set": 3,
                    "tie": 2,
                    "won": false,
                  },
                  Object {
                    "score": 6,
                    "set": 4,
                    "tie": null,
                    "won": true,
                  },
                  Object {
                    "score": 3,
                    "set": 5,
                    "tie": null,
                    "won": false,
                  },
                ],
              },
            }
          }
          displayLeftCircle="won"
          hasButton={true}
          isLive={false}
          isWatchable={false}
          liveButtonText="Live"
          matchInfoButtonText="Match Info"
          matchUrl="https://eurosport.com"
          schedule={null}
        />
      </Styled(div)>
    </ViewMore>
  </Styled(div)>
</Styled(div)>
`;
